rust/origen/cli/src/python.rs

```diff
@@ -34,7 +35,7 @@ impl Default for Config {
                     available = true;
                     let poetry_cmd = format!(
                         "{}/.poetry/bin/poetry",
-                        std::env::var("HOME").expect("$HOME is not defined")
+                        format!("{}", STATUS.home.to_string_lossy())
```

Comment:
Is there a reason for `to_string_lossy()` here and not `display()` as used elsewhere?
Could be, I'm just curious as I haven't came across this method yet.

Reply:
I found it in other code. I just needed a method that would satisfy the borrow checker. @coreyeng must be the one who uses that method.